common/scala/build.gradle

```diff
@@ -15,7 +15,7 @@ dependencies {
 
     compile 'com.typesafe.akka:akka-actor_2.11:2.4.16'
     compile 'com.typesafe.akka:akka-slf4j_2.11:2.4.16'
-    compile 'com.typesafe.akka:akka-http-core_2.11:10.0.9'
+    compile 'com.typesafe.akka:akka-http-core_2.11:10.0.10'
     compile 'com.typesafe.akka:akka-http-spray-json_2.11:10.0.2'
```

Comment:
Please update this related module as well. One should keep all akka deps on the same version to prevent version mismatches.

I got an open pr, which will facilitate this throughout the codebase: #2627